src/dom.js

```diff
@@ -670,6 +669,61 @@ const assertEquivalentNode = (actual, expected) => {
   }
 };
 
+/**
+ * prints dom in order for changes to be more discernible.
+ *
+ * @param {object} actual node from original page
+ * @param {object} expected node from test domain page
+ * @param {number} level current level in recursion tree
+ *
+ * return dump of dom that is indented at every level by level*2 spaces
+ */
+const recursiveDomDump = (actual, expected, level = 0) => {
```

Comment:
```suggestion
const dumpDOM = (actual, expected, level = 0) => {
```

- recursion is cool, but an implementation detail
- DOM is an acronym and should be capitalized